# key for defining mpc statistics (all in SI units)
    float32 sqp_failures
    float32 sqp_avg_time
    float32 sqp_max_time
    float32 agv_avg_lat_dev
    float32 agv_avg_speed